ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/panel/GamePanel.java

```diff
@@ -76,11 +81,34 @@ public GamePanel(@NotNull GlassPane glassPane, @NotNull Game game) {
     }
 
     private void initComponents() {
-        setLayout(new GridBagLayout());
+        setLayout(new BoxLayout(this, BoxLayout.LINE_AXIS));
 
         JPanel boardPanels = new JPanel();
         boardPanels.setOpaque(false);
-        boardPanels.setLayout(new GridBagLayout());
+        boardPanels.setLayout(new BoxLayout(boardPanels, BoxLayout.LINE_AXIS));
+        boardPanels.addComponentListener(new ComponentListener() {
+            @Override
+            public void componentResized(ComponentEvent e) {
+                mGameBoards[0].updateDimensions(e.getComponent().getWidth(), e.getComponent().getHeight());
```

Comment:
We need to do this for all boards, right?
`Stream.of(mGameBoards).forEach(...)`